tests/test_cv.py

```diff
@@ -177,3 +211,67 @@ def test_load_cv_folds_non_float_ids():
 
     # now read the CSV file using _load_cv_folds, which should raise ValueError
     _load_cv_folds(fold_file_path, ids_to_floats=True)
+
+def test_retrieve_cv_folds():
+    """
+    Test to make sure that the fold ids get returned correctly after cross-validation
+    """
+
+    learner = Learner('LogisticRegression')
+    num_folds = 5
+    cv_fs, custom_cv_folds = make_cv_folds_data(num_examples_per_fold=2, num_folds=num_folds)
+
+    expected_fold_ids = {'EXAMPLE_0': 0, 'EXAMPLE_1': 4, 'EXAMPLE_2': 3, 'EXAMPLE_3': 1,
```

Comment:
Isn't this the same as `custom_cv_folds`? Would it make more sense to use that as the expected output since that was what was used to create the input folds in the first place?